### DISCUSSION TITLE
Positioning of table cells

### DISCUSSION THREAD

**Discussion by callmeurpapa** [upptime/upptime#637] [Q&A]
Hi all, a question about the design of the WUI itself. Is it possible to make several columns in a table instead of one to increase visibility on the monitor?

**Comment by AnandChowdhary** [comment]
You can, for example, redesign the website using the same APIs as our static site generator.

**Comment by callmeurpapa** [comment]
WHere can i Find default design into my project?

**Reply by AnandChowdhary** [reply]
You can fork https://github.com/upptime/status-page :)
